pkg/Library/NodeGraph/NodeInspector.js

```diff
@@ -28,15 +28,15 @@ async update(inputs, state, {service, output, invalidate}) {
       // calling async method without awaiting it, on purpose,
       // which creates Special Circumstances, see below
       this.monitorStores(state, nodeTypes, {service, invalidate});
-    }
+    }      
```

Comment:
WS? My editor will delete it is all, so it will be in a commit one way or another :)

Reply:
fixed 
PS: what is "WS"?

Reply:
whitespace